pkg/nsm/config.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2021 Nordix Foundation
+Copyright (c) 2021-2024 Nordix Foundation
```

Comment:
```suggestion
Copyright (c) 2021 Nordix Foundation
Copyright (c) 2024 OpenInfra Foundation Europe
```

I think we should use OpenInfra Foundation Europe for copyright now